index.js

```diff
@@ -149,11 +149,23 @@ const getNpmCache = () => {
   return o
 }
 
+const getCypressCacheFolder = () => {
```

Comment:
@eplatek-safyr thank you for this change! this indeed fixes the issue for Macs but still leaves an issue for windows `/AppData/Local/Cypress/Cache`. Mind adding this as well?